libs/blocks/notification/notification.css

```diff
@@ -442,12 +453,17 @@
     gap: var(--spacing-m);
   }
 
-  .notification:is(.full-width, .ribbon) .foreground.container {
+  .notification:is(.full-width, .max-width-10-desktop) .foreground.container {
```

Comment:
Nit: Is there support for `max-width-6` or `-8` etc in this block? 

Also is there a reason for `.foreground.container` instead of just `.foreground` the appended classes can cause confusion unless theres a clear B.E.M treatment being used. Just curious if that was necessary?

Reply:
8 is the default for Banners (the default style of notification), 6 is not supported.

I am sorry about the `.container` addition, though -- in all honesty, this came from a time-saving measure as we were scrambling to get this done before shutdown. This all originally came from `aside.css`, and cherry-picking styles from that file made sense to me at the time. I can go through and remove it, but testing will take time.

Reply:
Wow, actually, I'm happy to report that just doing a find and replace for `.container` and removing all instances produced zero changes across the board. I triple checked against the exhaustive list of possible variants on the pages below on all breakpoints, but yeah honestly it looks like that worked just fine!

[Banners](http://ebartholomew/MWPW-155529-notification-parity--milo--adobecom.hlx.page/drafts/ebartholomew/notifications?martech=off)

[Ribbons](http://ebartholomew/MWPW-155529-notification-parity--milo--adobecom.hlx.page/drafts/ebartholomew/ribbon-notification?martech=off)

[Pills](http://ebartholomew/MWPW-155529-notification-parity--milo--adobecom.hlx.page/drafts/ebartholomew/pill-notifications?martech=off)